google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -1576,6 +1617,88 @@ public void testListResourceDescriptorAsyncWithOptions()
         Iterables.toArray(page.getValues(), MonitoredResourceDescriptor.class));
   }
 
+  @Test
+  public void testListLogs() {
```

Comment:
The test should explain what it tests. I mean, not only the fact that you test the API but also what behavior specifically is tested in the implementation. It should be either clear from the name or documented in comments. Similar comment for other tests

Reply:
I've followed the existing pattern here: simple name like this - "test functionality of list logs" just test a default behavior, same as with other methods. Special cases are also tested.
I agree that being more verbose helps. Having said that, should not it be done globally after we agree on the template?

Reply:
LMK if the pattern should be changed.

Reply:
A lot of this codebase (tests and library) was inherited. Being more verbose helps in maintenance. We should prioritize it when possible. Changing it across the entire codebase is desirable, but I don't want to hold your PR for something that's not related. If anything, we can open a low prio technical debt bug to improve it.